packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -10,11 +10,13 @@ import 'package:logging/logging.dart';
 import 'package:mime/mime.dart';
 import 'package:vm_service/vm_service.dart';
 
+import '../../screens/network/constants.dart';
 import '../../screens/network/network_model.dart';
 import '../globals.dart';
 import '../primitives/utils.dart';
 import 'http.dart';
 
+// ignore_for_file: avoid_dynamic_calls
```

Comment:
please remove this ignore and ensure we are casting JSON properly everywhere

Reply:
removed ignore, added cast